WiseBuy/View Controllers/BarcodeScanViewController.m

```diff
@@ -110,7 +109,7 @@ - (void)viewWillDisappear:(BOOL)animated {
     [self.captureSession stopRunning];
 }
 
--(void)captureOutput:(AVCaptureOutput *)output didOutputSampleBuffer:(nonnull CMSampleBufferRef)sampleBuffer fromConnection:(AVCaptureConnection *)connection{
+- (void)captureOutput:(AVCaptureOutput *)output didOutputSampleBuffer:(nonnull CMSampleBufferRef)sampleBuffer fromConnection:(AVCaptureConnection *)connection {
```

Comment:
nit: When we have a lot of params in a method, we usually try to put one param per line to make it more readable (like you have on line 131)